src/store/actions/actions.js

```diff
@@ -7,6 +7,78 @@ function updateCompose(compose) {
     };
 }
 
+function setRelease({ arch, distro }) {
+    return {
+        type: types.SET_RELEASE,
+        arch,
+        distro,
```

Comment:
A general convention with redux actions is for the format to be:
```
{
  type: `type name`
  payload: object
}
```
so here it would be:
```
{
  type: types.SET_RELEASE,
  payload: {
    arch,
    distro,
  }
}
```